RabbitMQ.Stream.Client/Message.cs

```diff
@@ -131,7 +131,6 @@ public static Message From(ref SequenceReader<byte> reader, uint len)
                         offset += AmqpWireFormatting.ReadAny(ref reader, out amqpValue);
                         break;
                     default:
-                        LogEventSource.Log.LogError($"dataCode: {dataCode} not handled. Please open an issue.");
```

Comment:
I have seen several times where a very-unexpected exception throws a message that includes a link to report an issue as well as important information to include with the issue. In this case it would be client library version, RabbitMQ version (if we have it), etc ... is that worth doing here or in another PR?

Reply:
Maybe another PR